connectors/katalog/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+ARG image=registry.access.redhat.com/ubi8/ubi-minimal:8.6
+FROM $image
```

Comment:
just from my curiosity, why do you need this image definition?

Reply:
to copy the ca.crt to the connector/manager pods in the new test. please see new target in Makfile that I mentioned above.

Reply:
if we use registry.access.redhat.com/ubi8/ubi-minimal we get the following error:
```
cd manager/testdata/notebook/read-flow-tls && ./copy-cacert-to-pods.sh
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "0f2cb3310b17a6a86161203a356de6ad2153c83db76c7d7a3b7342260fb19cdd": OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "tar": executable file not found in $PATH: unknown
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "d5e4842144d7b5ed4a60c5fc15caa1851b15fdd634ecf276958dd47cc23bcbbe": OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "tar": executable file not found in $PATH: unknown
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "aa197e8a48097d145901326b658f3eb4baac2b518a467c987300d3f42fa5d4f2": OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "tar": executable file not found in $PATH: unknown
Makefile:142: recipe for target 'run-notebook-readflow-tls-system-cacerts-tests' failed
make: *** [run-notebook-readflow-tls-system-cacerts-tests] Error 1